### PR TITLE
Added test files and changed settings for Verify command

### PR DESCRIPTION
**Pull request by barthanssens**
Signed-off-by: Bart Hanssens <bart.hanssens@bosa.fgov.be>

This PR addresses GitHub issue: #41  .

Briefly describe the changes proposed in this PR:

* Added some test files
* Changed deprecated verification settings to more general settings
* Write warnings/non-fatal errors as errors to the console (which seems to be the point of the command) instead of just a notice
* Treat RDFParseException as other exceptions (i.e. write error to console)

### CODE REVIEW

File: console/src/main/java/org/eclipse/rdf4j/console/VerificationLIstener.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * 
  * @author Dale Visser
  */
-class VerificationListener extends AbstractRDFHandler implements ParseErrorListener {
+class VerificationLIstener extends AbstractRDFHandler implements ParseErrorListener {
```
Comment (jeenbroekstra) [inline console/src/main/java/org/eclipse/rdf4j/console/VerificationLIstener.java]: Typo?

**Comment by jeenbroekstra** [comment]
@barthanssens you available to fix the typo and resolve the conflicts?

**Comment by barthanssens** [comment]
Ok, I'll take a look at it

**Comment by barthanssens** [comment]
@jeenbroekstra  after a few changes the build now seems to fail with 

> Could not resolve dependencies for project org.eclipse.rdf4j:rdf4j-server-compliance:war:2.5-SNAPSHOT: The following artifacts could not be resolved: org.eclipse.rdf4j:rdf4j-store-testsuite:jar:2.5-SNAPSHOT,

**Comment by jeenbroekstra** [comment]
@barthanssens caused by Jenkins builds falling over due to the messy release cycle for 2.4.0-M1. I've restarted the Jenkins builds that deploy these snapshots, after that Travis should be able to pick them up.

TL;DR it's not you it's me.

**Comment by jeenbroekstra** [comment]
Oh dear. I just realized you synced this with the develop branch _after_ I prepped that branch for the next release (2.5).

Do you think you can rebase your changes onto the master branch (or cherry-pick perhaps) so they can be merged in there before we release 2.4.0? Sorry for the extra work.

**Comment by jeenbroekstra** [comment]
Forget my earlier comment about rebasing/cherry-picking, just realized I can very easily just update the pull request itself in GitHub. Rebased the PR against master so it can be included in the 2.4 release.
